pkg/detectors/twitch/twitch_test.go

```diff
@@ -47,14 +49,87 @@ func TestTwitch_FromChunk(t *testing.T) {
 				data:   []byte(fmt.Sprintf("You can find a twitch secret %s within twitch %s", secret, id)),
 				verify: true,
 			},
+			// the detector will try every combination of the secret and id for
+			// client_id and client_secret, so we expect 4 results
+			// but only 1 of them will be verified
 			want: []detectors.Result{
+				{
+					DetectorType: detectorspb.DetectorType_Twitch,
+					Verified:     false,
+				},
 				{
 					DetectorType: detectorspb.DetectorType_Twitch,
 					Verified:     true,
 				},
+				{
+					DetectorType: detectorspb.DetectorType_Twitch,
+					Verified:     false,
+				},
+				{
+					DetectorType: detectorspb.DetectorType_Twitch,
+					Verified:     false,
+				},
```

Comment:
is the order of these results consistent? i think that the test will flake if not.

Reply:
yes it should always consistent bc the data string doesn't change.